### PR TITLE
Changing config to more json attributes and more standard values Fixes #49

### LINKED ISSUE #49: Refactor objects for each column in config.json
Parsing the `method` property in each table object in `config.json` currently relies on splitting on `.` and `__`. We could probably nest under the property more elaborate objects, which would simplify the code and make it more readable.

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,92 +1,92 @@
 {
     "academic_terms": [
-        {"name": "id", "method": "None"},
-        {"name": "canvas_id", "method": "None"},
-        {"name": "name", "method": "None"},
-        {"name": "date_start", "method": "None"},
-        {"name": "date_end", "method": "None"}
+        {"name": "id", "module": "None"},
```
Comment (ssciolla) [inline config.json]: Could we use `null` instead of "None"?

Comment (jonespm) [reply config.json]: Yeah I think that could work!

**Comment by jonespm** [comment]
Yeah, I'm not sure what the best way to handle this. 
I made a push so everything has a module and method now. The ones that didn't were in the local "util_methods" which is what they should have been.

I think this comes a lot closer to aligning to you idea now looking at the code. Really some methods are going to have custom kwargs that others don't have. I don't think I'm 100% ready to make this change yet as there's a little more work needed.

Everything that operates on the entire column is currently passing in the dataframe and parameters. These will be easy to add

Everything that operates on a cell still has some special cases. I feel like we should maybe defer that stuff to another ticket.

It looks like these recent commits from your review are still working well for me.

File: mylasqlanon.py [hunk 2]
```diff
@@ -127,21 +130,27 @@
...
     for col in t_config:
-        mod_name, index_name = (col.get('method').rsplit('.', 1) + [None] * 2)[:2]
-        col_name = col.get('name')
+        mod_name = col.get("module")
+        method_name = col.get("method")
+        index_name = col.get("index")
+        col_name = col.get("name")
+        source_name = col.get("source")
+        if mod_name and method_name:
+            dyn_func = getattr(locals().get(mod_name), method_name)
```
Comment (ssciolla) [inline mylasqlanon.py]: This dynamic function idea is cool, but perhaps makes the code a little more difficult to read. Maybe we could just add a debug log message somewhere here that provides the function name?

**Pull request merged**